libs/client-api-for-wasm/src/entities.rs

```diff
@@ -0,0 +1,79 @@
+use client_api::error::ErrorCode;
+use serde::{Deserialize, Serialize};
+use tsify::Tsify;
+use wasm_bindgen::JsValue;
+
+#[derive(Tsify, Serialize, Deserialize, Default, Debug)]
+#[tsify(into_wasm_abi, from_wasm_abi)]
+pub struct Configuration {
```

Comment:
Can we reuse the structs including Config, ErrorCode etc. in client-api directly? then we don't need to define again.

Reply:
Then we need to add
#[derive(Tsify, Serialize, Deserialize, Default, Debug)]
#[tsify(into_wasm_abi, from_wasm_abi)]
to these structs
I think it will make dirty

Reply:
How about using cfg feature flag? Only enable when current target is wasm